docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,55 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_NETWORK: ${NBITCOIN_NETWORK:-regtest}
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        deprecatedrpc=signrawtransaction
+        addresstype=p2sh-segwit
+        rpcuser=ceiwHEbqWI83
```

Comment:
remove this as well

Reply:
Eclair needs it

Reply:
1. eclair do not support cookie auth?
2. don't use rpcuser/password, instead use rpcauth. (like I did for lnd)
3. I don't know as I don't know if you can have more than 1 user per bitcoin instance, and one is created for lnd in the bitcoin image.

Reply:
@araspitzu can probably answer this better

Reply:
If Eclair support cookiefile, it is really important that eclair actually read the file before making call, and do not cache the value, as bitcoind can restart and change the content of the cookiefile.

Reply:
So I checked: Bitcoin core support multiple rpcauth.

So you just need to convert rpcuser/pass to rpcauth like I did in bitcoin-lnd (you need to pull, I just made new commit for this)

You can use NBitcoin for that RPCClient.GetRPCAuth or something like this.

Reply:
Eclair only supports user/password authentication towards bitcoin, it also caches those values from the configuration so if you change them in bitcoin you need to update eclair.conf and restart eclair.

Reply:
I switch to rpcauth